core/src/test/resources/io/grpc/internal/test_retry_service_config.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
It's better to use different files (at least copies with different file names) for general service config test vs retry test, because future changes on the data for one of the tests won't affect the other.